tools/src/h5diff/ph5diff_main.c

```diff
@@ -85,8 +85,9 @@ main(int argc, char *argv[])
 
             MPI_Barrier(MPI_COMM_WORLD);
 
-            print_info(&opts);
             print_manager_output();
+
+            print_info(&opts);
```

Comment:
Make the lines printed by `print_info` come at the bottom of output, similar to serial `h5diff`